mentii/mentii
# Enable prod mode on staging

Enables angular's prod mode on the staging server and prod

**Significant Changes**
1. Created a new function in mentii config to return true if the code is deployed on either prod or staging.
2. Use this new function to enable prod mode in main.ts

**How to Test**
1. _Once code is deployed_
2. Go to staging and prod
3. Open the developer console in chrome.
4. There should be no message about enabling prod mode
5. Run it locally and you should see this message.

**Notes**
Have a gret day
